test/integration/events/events_test.go

```diff
@@ -84,9 +86,15 @@ func TestEventCompatibility(t *testing.T) {
 			return false, err
 		}
 
-		if len(events.Items) != 2 {
+		if len(events.Items) < 2 {
 			return false, nil
 		}
+
+		// Be sure that both APIs produce the same number of events.
+		if len(events.Items) != len(v1Events.Items) {
```

Comment:
This looks racy, since time elapsed between the two GET calls.

I suggest instead using a unique namespace.

Reply:
> This looks racy, since time elapsed between the two GET calls.
> 
> I suggest instead using a unique namespace.

Racier in a pre-existing way or racier in a novel way?

Reply:
> I suggest instead using a unique namespace.

My read of the test is that they're using the kube-apiserver as known event emitter and that makes the set of events emitted to reach steady-state predictable and known to be greater than 1.